.mprocs.yaml

```diff
@@ -0,0 +1,21 @@
+procs:
+  user:
+    shell: source scripts/tmux-user-shell.sh
```

Comment:
does this work?

Reply:
Maybe? I only see a `Waiting fedimintd to start` message in this shell

Reply:
yes, that should be solved after you spawn the dev federation.

But does this give an interactive shell, try commenting out tmux-user-shell for now

Reply:
Okay, will do

Reply:
Yep, using `bash` inplace of running `tmux-user-shell.sh` does indeed give us interactive shell, inside the nix environment.